GenderPayGap.WebUI/Controllers/Account/ChangeContactPreferencesController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.WebUI.Classes;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Models.Account;
+using GovUkDesignSystem.Parsers;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GenderPayGap.WebUI.Controllers.Account
+{
+    [Route("manage-account")]
+    public class ChangeContactPreferencesController : Controller
+    {
+
+        private readonly IDataRepository dataRepository;
+
+        public ChangeContactPreferencesController(
+            IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+        
+        // The user asks to change their contact preferences
+        // We direct them to a page that asks for their updated contact preferences
+        [HttpGet("change-contact-preferences")]
+        public IActionResult ChangeContactPreferencesGet()
+        {
+            var viewModel = new ChangeContactPreferencesViewModel();

```

Comment:
This view-model isn't used until a few lines below, so I'd be tempted to move it down below the next line...